src/main/java/com/cloudbees/jenkins/support/SupportAction.java

```diff
@@ -588,9 +593,8 @@ public void doDownloadBundle(@QueryParameter("taskId") String taskId, StaplerRes
 
         rsp.setContentType("application/zip");
         rsp.addHeader("Content-Disposition", "attachment; filename=" + supportBundleName);
-        try (ServletOutputStream outputStream = rsp.getOutputStream();
-                FileInputStream inputStream = new FileInputStream(bundleFile)) {
-            IOUtils.copy(inputStream, outputStream);
+        try (ServletOutputStream outputStream = rsp.getOutputStream()) {
```

Comment:
stylistically, just
```suggestion
        try (OutputStream outputStream = rsp.getOutputStream()) {
```
or
```suggestion
        try (var outputStream = rsp.getOutputStream()) {
```
suffices